strings-csharp/GetFirstNCharactersOfAString/GetFirstNCharactersOfAString/FirstNCharactersOfStringGetter.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace GetFirstNCharactersOfAString;
+public class FirstNCharactersOfStringGetter(string inputString = "CodeMaze", int numberOfCharacters = 4)
+{
+    private readonly string _inputString = inputString;
+    private readonly int _numberOfCharacters = numberOfCharacters;
+
+    private bool IsInputStringNullOrEmptyOrShorter()
+        => string.IsNullOrEmpty(_inputString) || _inputString.Length < _numberOfCharacters;
+
+    public string? UseRemove()
```

Comment:
This applies throughout the class, since we are guaranteed that `_inputString` is not null, then the return value of these methods is also guaranteed to never be null. No need then for `string?` as the return type.